### PR TITLE
feat:Add Starry Net tutorial

### PR DESCRIPTION
**Pull request by sathomas2**
<br>
Adds a tutorial for running the Starry Net forecasting pipeline.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/forecasting/starry_net_pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,1085 @@
+{
```
Comment (gericdong) [inline notebooks/official/forecasting/starry_net_pipeline.ipynb:1]: <p>Tensorboard &gt; TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

[the remark this answers is not in the document]
Comment (sathomas2) [reply notebooks/official/forecasting/starry_net_pipeline.ipynb:1]: <p>Yikes, i forgot to update them after moving to official directory. Done.</p>

Comment (sathomas2) [reply notebooks/official/forecasting/starry_net_pipeline.ipynb:1]: <p>Done.</p>

**Review by gericdong — APPROVED** [review]

**Pull request merged**